spec/index.html

```diff
@@ -1174,23 +1174,67 @@ <h5>DELETE WHERE</h5>
 &lt;http://example/william&gt; foaf:mbox  &lt;mailto:bill@example&gt; .</pre>
           </section>
         </section>
+
         <section id="load">
           <h4>LOAD</h4>
-          <p>The <code>LOAD</code> operation reads an RDF document from a IRI and inserts its triples into the specified graph in the Graph Store. The specified destination graph <em class="rfc2119"
-          title="Keyword in RFC 2119 context">SHOULD</em> be created if required; again, implementations providing an update service over a fixed set of graphs <em class="rfc2119" title=
-          "Keyword in RFC 2119 context">MUST</em> return with failure for a request that would create a disallowed graph. If the destination graph already exists, then no data in that graph will be
-          removed.</p>
-          <pre class="defn nohighlight"><strong>LOAD</strong> ( <strong>SILENT</strong> )? <em><a data-cite=
-          "SPARQL12-QUERY#rIRIREF">IRIref_from</a></em> ( <strong>INTO GRAPH</strong> <em><a data-cite="SPARQL12-QUERY#rIRIREF">IRIref_to</a></em> )?</pre>
-          <p><em>IRIref_from</em> specifies the IRI of a document such that a store will be able to identify, locate and read the document. The most common form will be URLs with the <em>http</em>
-          IRI schemes. Once the document has been read, the resulting triples will be inserted into the destination graph named by the IRI referred to by <em>IRIref_to</em>.</p>
-          <p>If no destination graph IRI (<em>IRIref_to</em>) is provided to load the triples into, then the data will be loaded into the default graph.</p>
-          <p>In case no RDF data can be retrieved (as opposed to the empty graph being retrieved) from the IRI denoted by <em>IRIref_from</em>, or in case the retrieval method returns an error (such
-          as, for instance an HTTP error code), the SPARQL 1.2 Update service <em class="rfc2119" title="Keyword in RFC 2119 context">SHOULD</em> return failure and the status of the Graph Store
-          <em class="rfc2119" title="Keyword in RFC 2119 context">SHOULD</em> remain in the same status as prior to the request; in case the keyword SILENT is present, however, the operation will
-          still return success and the status of the Graph Store is not specified by the present document: implementations may create the destination graph or not and partially load data, in case of
-          a transmission error where partial data has been received (which itself may be legal RDF).</p>
+
+          <p>
+            The `LOAD` operation reads an RDF document from a IRI and inserts triples
+            into the graphs of the dataset in the  <a href="#defn_graphStore">Graph Store</a>.
```

Comment:
```suggestion
            The `LOAD` operation reads an RDF document from a IRI and inserts triples therefrom
            into the graphs of the dataset in the  <a href="#defn_graphStore">Graph Store</a>.
```

Reply:
"therefrom" is archaic at [least in the OED](https://www.google.com/search?q=define%3A+therefrom) (public access).


and [declining](https://books.google.com/ngrams/graph?year_start=1800&year_end=2019&corpus=26&smoothing=7&case_insensitive=on&content=therefrom

:smile:

Do you have another suggestion?

Reply:
```suggestion
            The `LOAD` operation reads an RDF document from an IRI and inserts triples from the document
            into the graph(s) of a dataset in a <a href="#defn_graphStore">Graph Store</a>.
```

Reply:
Reworded to be as neutral as possible. (We can't talk about quads.) 

```html
The `LOAD` operation reads an RDF document from an IRI and inserts triples
from the document into the <a href="#defn_graphStore">Graph Store</a>.
```

Section 2 is "The Graph Store" and this runs through the document.

The destination within the graph store is described later.

Reply:
Works for me. I don't have a "resolved" button, but if you do, please press it.